linode/manager
# change: [M3-8209] - Update Region label globally

## Description 📝
Follow-up to https://github.com/linode/manager/pull/10702 except without the feature flagging due to additional complexity for testing. This PR also includes state in the label for regions in the US

## Changes  🔄
List any change relevant to the reviewer.
- Display Country, State, City for regions in the US
- Display Country, City for all other regions

## Target release date 🗓️
8/19. This change will be rolled out globally and separate from Gecko (approved by Lew)

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![image](https://github.com/user-attachments/assets/daf103f4-cbef-451a-8726-58c0acddff8b) | ![image](https://github.com/user-attachments/assets/bec6fff3-bdff-4b58-b7bb-9bfa5e8ab21c) |
| ![image](https://github.com/user-attachments/assets/6664a512-4a56-483a-a591-023793130b03) | ![image](https://github.com/user-attachments/assets/ff7bf16a-36be-47a5-9368-5e8556b03609) |

## How to test 🧪
### Verification steps
(How to verify changes)
- Add  `gecko2: {
      enabled: false,
    },` to `useFlags.ts` to turn Gecko features off locally
  - We do not want to change the LD flag since other teams rely on it
- Check region labels throughout the app. The label should display as Country, State, City for regions in the US and Country, City for other regions.

## As an Author I have considered 🤔

*Check all that apply*
